client/app/src/main/res/layout/activity_my_book.xml

```diff
@@ -18,22 +18,23 @@
         android:layout_marginBottom="16dp"
         android:background="@drawable/rounded_button"
         android:text="@string/edit_book"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        android:textAllCaps="false"/>
+        android:textAllCaps="false"
+        app:layout_constraintBottom_toTopOf="@+id/bottom_navigation_frag"
+        app:layout_constraintEnd_toEndOf="parent" />
 
     <Button
         android:id="@+id/see_requests_button"
-        android:layout_width="120sp"
+        android:layout_width="0dp"
```

Comment:
Does this change the layout?

Reply:
The width is based on the distance from the other two buttons at this point. So the layout isn't changed and it'll work better with smaller screen sizes.

Reply:
Ah I see, okay, looks good !